kong/templates/nginx_kong.lua

```diff
@@ -209,8 +209,8 @@ server {
 > end
 
     # injected nginx_admin_* directives
-> for k, v in pairs(nginx_admin_directives)  do
-    $(k) $(v);
+> for _, el in pairs(nginx_admin_directives)  do
```

Comment:
We should also switch the iterator to `ipairs` - again, the benefit here is that we can guarantee the order of insertion of each directive across architectures (or even sort it in the future, or guarantee the same order as that of insertion in the configuration file, etc...).